Assets/VRTK/Scripts/VRTK_InteractTouch.cs

```diff
@@ -401,6 +401,7 @@ private void CreateTouchCollider()
                 {
                     controllerCollisionDetector = Instantiate(customRigidbodyObject, transform.position, transform.rotation) as GameObject;
                     controllerCollisionDetector.transform.SetParent(transform);
+                    controllerCollisionDetector.transform.localScale = transform.localScale;
```

Comment:
hummmmmmmmmmmm, if it's custom should you be doing anything automatic to it?

what if you wanted to provide a custom collider and specifically have it smaller (or the size you created it at)?